src/NewCard/Header.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import defaultTheme from "../defaultTheme";
+import ChevronDown from "../icons/ChevronDown";
+import Stack from "../Stack";
+import Heading from "../Heading";
+import Text from "../Text";
+
+const ChevronIcon = styled(ChevronDown)`
+  transform: ${({ expanded }) => expanded && "rotate(-180deg)"};
+  transition: transform ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+`;
+
+ChevronIcon.defaultProps = {
+  theme: defaultTheme,
+};
+
+type Props = {|
+  description?: React.Node,
+  icon?: React.Node,
+  actions?: React.Node,
+  title?: React.Node,
+  dataA11ySection?: string,
+  expandable?: boolean,
+  expanded?: boolean,
+|};
+
+const Header = ({
+  description,
+  icon,
+  title,
+  actions,
+  dataA11ySection,
+  expandable,
+  expanded,
+}: Props) => (
+  <>
+    <Stack align="center" spacing="compact" spaceAfter="smallest">
```

Comment:
This produces unwanted `margin-bottom` when there is only `Header`.
<img width="640" alt="Snímek obrazovky 2019-11-26 v 13 22 41" src="https://user-images.githubusercontent.com/7254437/69633093-f5c4d600-104f-11ea-81ec-acc6d5ecf223.png">

Reply:
The same is when used in `CardSection`
<img width="646" alt="Snímek obrazovky 2019-11-26 v 13 25 09" src="https://user-images.githubusercontent.com/7254437/69633391-666bf280-1050-11ea-8f6e-eee4f41b817c.png">